spec/support/features/document_capture_step_helper.rb

```diff
@@ -11,9 +11,9 @@ def attach_and_submit_images
     submit_images
   end
 
-  def attach_images(file = 'app/assets/images/logo.png')
-    attach_file t('doc_auth.headings.document_capture_front'), file
-    attach_file t('doc_auth.headings.document_capture_back'), file
+  def attach_images(file = Rails.root.join('app', 'assets', 'images', 'logo.png'))
+    attach_file t('doc_auth.headings.document_capture_front'), file, make_visible: true
+    attach_file t('doc_auth.headings.document_capture_back'), file, make_visible: true
```

Comment:
I may have some concern about forcing the input to be visible if it hides some underlying issue with the input being available. That being said, I'm pretty sure the input is heavily stylized and perhaps even hidden in the default experience anyways, so it may not be worth concerning too much over.

Reply:
This is just a temp change to work around the fact that file input fields are often hidden to make a more appealing presentation. From the [docs](https://www.rubydoc.info/github/teamcapybara/capybara/Capybara/Node/Actions#attach_file-instance_method):

>  In the case of the file field being hidden for styling reasons the make_visible option can be used to temporarily change the CSS of the file field, attach the file, and then revert the CSS back to original.